src/main/java/org/skriptlang/skript/lang/script/Script.java

```diff
@@ -38,6 +39,7 @@
  * Every script also has its own internal information, such as
  *  custom data, suppressed warnings, and associated event handlers.
  */
+@ApiStatus.Internal
```

Comment:
Why is this internal? It should be allowed public. It's the main access to Script data which structures use.

Reply:
Oh I meant to put this on the constructor